tests/test_wallet.py

```diff
@@ -1045,6 +1045,36 @@ def test_hsmtool_secret_decryption(node_factory):
     assert node_id == l1.rpc.getinfo()["id"]
 
 
+@unittest.skipIf(TEST_NETWORK == 'liquid-regtest', '')
```

Comment:
Add a comment saying that this is only supported for `mainnet` and `testnet` (technically we'd require it to be `regtest` but that appears to match `testnet`).

Reply:
Yeah descriptors for regtest are the same than for testnet (but addresses the same as mainnet!! :sob: )

Reply:
Hurray for consistency 🤦